package.json

```diff
@@ -43,7 +43,7 @@
     "prettycron": "0.10.0",
     "query-string": "4.1.0",
     "react": "0.14.7",
-    "react-ace": "3.4.1",
+    "react-ace": "3.7.0",
```

Comment:
Yes, there is a big change on the `4.x` branch. `3.x` is still compatible like before. In addition we now have a single place the ACE editor is used (the `JSONEditor` component) so if something goes wrong, that's the only place we have to touch.